webservice/src/test/java/pl/edu/pjatk/lnpayments/webservice/admin/service/AdminServiceTest.java

```diff
@@ -155,4 +158,59 @@ void shouldThrowExceptionWhenUserAlreadyHasKeys() {
                 .withMessage("User has already uploaded his keys!");
     }
 
+    @Test
+    void shouldRemoveUserCorrectly() {
+        AdminRequest request = new AdminRequest("test@test.pl", "test", "pass");
+        AdminUser expectedUser = new AdminUser("test@test.pl", "test", "pass");
+        AdminDeleteRequest getEmail = new AdminDeleteRequest("test@test.pl");
+
+        adminService.createAdmin(request);
+        when(adminUserRepository.findByEmail(request.getEmail())).thenReturn(Optional.of(expectedUser));
```

Comment:
To samo tutaj, testujemy usuwanie, nie dodawanie. Fajnie by też było jakbyś grupował instrukcje zgodnie z zasadą AAA.

Reply:
poprawiłem